modules/era/sql/synth_recipes_era.sql

```diff
@@ -18,453 +18,3476 @@ ALTER TABLE `synth_recipes`
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Scorpion Subligar' AND ID = 53020;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Scutum' AND ID = 12547;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Scythe' AND ID = 11512;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Seadog Gun' AND ID = 53023;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sha\'ir Gages' AND ID = 44537;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sha\'ir Turban' AND ID = 34503;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Shadow Bow' AND ID = 44041;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Shadow Roll' AND ID = 31536;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Shell Lamp' AND ID = 54523;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Shining Ring' AND ID = 24552;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Shinobi Gi' AND ID = 33563;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Shinobi Hachigane' AND ID = 33501;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Shinobi Hakama' AND ID = 33558;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Shinobi Kyahan' AND ID = 33524;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Shinobi Tekko' AND ID = 33534;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Shiny Gold Thread' AND ID = 32533;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Shock Absorber' AND ID = 22521;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Shock Subligar' AND ID = 52501;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Shrimp Cracker' AND ID = 73047;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Shrimp Cracker' AND ID = 73048;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Silencing Potion' AND ID = 60031;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Silk Hat' AND ID = 33022;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Silk Slacks' AND ID = 33525;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Silken Coat' AND ID = 33040;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sipahi Boots' AND ID = 23529;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sipahi Dastanas' AND ID = 44046;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sipahi Jawshan' AND ID = 44504;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sipahi Turban' AND ID = 34030;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sis Kebabi' AND ID = 72558;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Smash Cesti' AND ID = 40525;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Smooth Beetle Jaw' AND ID = 51010;
```

Comment:
This is a Mist set crafting item. Unable to find an exact date for it, but comments in the materials suggest late Zilart or Early COP
http://web.archive.org/web/20201127042454/https://ffxi.allakhazam.com/db/item.html?fitem=5660
http://web.archive.org/web/20050207021924/http://ffxi.allakhazam.com:80/db/item.html?fitem=5785